app/models/reward.rb

```diff
@@ -12,4 +12,19 @@ class Reward < ApplicationRecord
   has_many :categories, through: :category_rewards
 
   has_one_attached :photo
+
+  def self.import(file)
+    ActiveRecord::Base.transaction do
+      CSV.foreach(file.path, headers: true) do |row|
+        if row[3].nil?
+          Reward.create_with(description: row[1], price: row[2]).find_or_create_by!(title: row[0], slug: row[0].parameterize)
```

Comment:
you should find a reward only by its slug. What if an admin updates the title of a reward? :)

Reply:
I changed it :)